### DISCUSSION TITLE
Install on Mac Issues

### DISCUSSION THREAD

**Discussion by jonlax** [Le0Michine/FusionGridfinityGenerator#32] [Q&A] [answered]
I am banging my head against the wall trying to get the install on my Mac. 

- Fusion 360 2.0.16486 x86_64 [Rosetta 2] 
- Have tried multiple times to add it in
- Does not show up in menu

I placed the dir FusionGridfinityGenerator-master inside LibraryApplication Support/Autodesk/Autodesk Fusion 360/API/

What am I doing wrong?

**Comment by Le0Michine** [comment, marked as answer]
Hi, I'm excited to share that it is now available vis Autodesk App Store which includes automated installer for MacOS.
https://apps.autodesk.com/FUSION/en/Detail/Index?id=7197558650811789 
Hope it will resolve the issue for you

**Reply by jonlax** [reply]
This worked! Thank you.
